nilearn/datasets/utils.py

```diff
@@ -489,13 +509,12 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
     password: string, optional
         Password used for basic HTTP authentication
 
-    handlers: list of BaseHandler, optional
-        urllib handlers passed to urllib.request.build_opener. Used by
-        advanced users to customize request handling.
-
     verbose: int, optional
         verbosity level (0 means no message).
 
+    session:
+        requests Session to use to send requests
```

Comment:
Should we reference the requests docs, here ? Thinking in particular of: https://requests.readthedocs.io/en/master/user/advanced/#session-objects

Reply:
sure!

Reply:
actually I just made the import more explicit (`requests.Session`) but I'd rather not add the link as it is likely to become stale at some point and the requests documentation (online or in docstrings) is very easy to find